cli/bench/main.rs

```diff
@@ -314,7 +314,10 @@ fn get_binary_sizes(target_dir: &Path) -> Result<HashMap<String, i64>> {
 
 const BUNDLES: &[(&str, &str)] = &[
   ("file_server", "./test_util/std/http/file_server.ts"),
-  ("gist", "./test_util/std/examples/gist.ts"),
+  (
+    "gist",
+    "https://github.com/denoland/denobyexample/raw/main/data/hello-world.ts",
```

Comment:
I think this is an overkill to request a remote module that has a single `console.log`. Could you please add this file to `cli/tests/testdata/` and use it instead?